test/e2e/quick_start_test.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 var _ = Describe("Quick start", Serial, func() {
-	for _, provider := range []string{"Docker", "AWS"} {
+	for _, provider := range []string{"Docker", "AWS", "Nutanix"} {
```

Comment:
I think we want to factor this out because Nutanix don't support `"Calico"` or `"ClusterResourceSet"`